orchagent/orchdaemon.cpp

```diff
@@ -864,7 +864,13 @@ void OrchDaemon::start()
                     flush();
 
                     SWSS_LOG_WARN("Orchagent is frozen for warm restart!");
-                    sleep(UINT_MAX);
+                    while (true)
+                    {
+                        // Send heartbeat message to prevent Orchagent stuck alert.
```

Comment:
Just curious - at some point in time in the shutdown process, swss will be down. At that time will there still be OA stuck messages?

Reply:
This message will send to systemd, then systemd will forward to process-exit-listener. 

process-exit-listener will monitor both heartbeat message and orchanget process exit message.

If the orchange is down, process-exit-listener will receive the process exit signal, then send process not running warning. in that case there will be no OA stuck message.